amy/templates/navigation.html

```diff
@@ -82,6 +82,7 @@
               <a class="dropdown-item" href="{% url 'autoupdate_profile' %}">Your profile</a>
               {% endif %}
               <a class="dropdown-item" href="{% url 'api:export-person-data' %}?format=json">Download your data</a>
+              <a class="dropdown-item" href="{% url 'person_archive' user.id %}?format=json">Archive your profile</a>
```

Comment:
`person_archive` returns JSON?

Reply:
Oh good catch. no it does not.

Reply:
Does it respond to GET requests?

Reply:
Huh... well it probably shouldn't but apparently does. Archiving from that button does work. I'm not sure why though